### ISSUE TITLE
Images built on windows have windows style directory separators \

### ISSUE THREAD

**Issue by dcherniv** [GoogleContainerTools/skaffold#695]
### Expected behavior
  docker image to have / as the directory separator for the context copied from local.
### Actual behavior
  docker image has \ as the directory separator, directory structure is not created, instead we have long files of this sort: src\java\main\test.java
### Information
skaffolv 0.7.0
windows
### Steps to reproduce the behavior

1. install skaffold 0.7.0
2. have an ADD or COPY in your dockerfiles.
3. skaffold dev

Fix is attached https://github.com/GoogleContainerTools/skaffold/pull/693

**Comment by dlorenc** [comment]
Looks like this introduced some test failures:

```
TestDockerContext (0.00s)
	context_test.go:81: File files/included.txt should have been included, but was not
	context_test.go:84: File Dockerfile should have been included, but was not
```

**Comment by dcherniv** [comment]
@dlorenc interesting. the test suite runs on linux? i will test on linux later today.
But the behavior i observed on windows is most definitely incorrect, with the forward slashes going into the tar.

**Comment by dcherniv** [comment]
@dlorenc please check again #693. 
Side note, there's some inconsistencies with parameter positions.  Also camel case/pascal case
E.G.
in docker/parse.go
```
func readDockerfile(workspace, dockerfilePath string) ([]string, error) {
func GetDependencies(dockerfilePath, workspace string) ([]string, error) 
```

**Comment by dlorenc** [comment]
I beliee this was fixed with #693
